conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureExceptionHandler.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (c) Copyright 2018 Palantir Technologies Inc. All rights reserved.
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
```

Comment:
nit: I believe our stance is to avoid updating copyright date.

Reply:
The diff is a little misleading, the old version moved to ConjureExceptionHandlerHttpHandler which still has a 2018 header, and I reused this name for the Implementation of ExceptionHandler. Handler is overloaded between exchange handlers (HttpHandler) and exception handlers.